pkg/cmd/generate/templates.go

```diff
@@ -2,34 +2,68 @@ package generate
 
 import (
 	"github.com/MakeNowJust/heredoc"
+	"github.com/redhat-developer/app-services-cli/pkg/cmd/registry/registrycmdutil"
 )
 
 var (
 	templateEnv = heredoc.Doc(`
-		## Generated by rhoas generate-config
-		{{if .KafkaHost}}KAFKA_HOST={{.KafkaHost}} 
-		{{end}}{{if .RegistryURL}}SERVICE_REGISTRY_URL={{.RegistryURL}}
-		{{end}}RHOAS_CLIENT_ID={{.ClientID}}
+		## Generated by rhoas cli
+		{{if .KafkaHost}}## Kafka Configuration
+		KAFKA_HOST={{.KafkaHost}}
+		{{end}}{{if .RegistryURL}}## Service Registry Configuration
+		SERVICE_REGISTRY_URL={{.RegistryURL}}
+		SERVICE_REGISTRY_CORE_PATH=` + registrycmdutil.REGISTRY_CORE_PATH + `
+		SERVICE_REGISTRY_COMPAT_PATH=` + registrycmdutil.REGISTRY_COMPAT_PATH + `
```

Comment:
Will it be better to provide the compatibility endpoint at once? As a user, I think I'll prefer to use one single env var over appending them myself in app.

Reply:
Ignore. Looks okay after seeing the [PR](https://github.com/redhat-developer/app-services-guides/pull/469) in guides repo.

Reply:
We need them separate as some tools require full uri and some just hostname